### DISCUSSION TITLE
S3 Minio + Dify

### DISCUSSION THREAD

**Discussion by Srharsha0006** [langgenius/dify#5300] [Help]
I have configured the MinIO details in the Dify docker-compose.yaml file, and the application is running and working fine. I want to see the files in MinIO from Dify. How can I search the files in the MinIO bucket from the Dify web interface?

**Comment by perzeuss** [comment]
You cannot search files in the S3 bucket from Dify. You need to use other tools for that.

**Comment by Yaya2344** [comment]
have you solve this problems?can you tell me how to solve it please

**Comment by DavideDelbianco** [comment]
While you cannot use Dify to search the files, MinIO exposes a web interface on port 9001 

![image](https://github.com/user-attachments/assets/05c4ae18-572f-4f88-80db-16d1f4ca5590)
